### DISCUSSION TITLE
Not able to connect Matter devices to ZBDongle-E

### DISCUSSION THREAD

**Discussion by Solarflor** [darkxst/silabs-firmware-builder#77] [Q&A]
I have flashed the MultiPAN RCP to my ZBDongle-E to manage both ZigBee and Thread systems.

After installing all the required integrations the following repair request appeared on my Home Assistant.

"OTBR and ZHA share the same radio but use different channels.
When OTBR and ZHA share the radio, they must use the same network channel.
If OTBR and ZHA try to connect to networks on different channels, neither Thread/Matter nor Zigbee will work.
OTBR is configured with a Thread network on channel 15, ZHA is configured with a Zigbee network on channel 25."

When I tryed to change the channel of Thread, the following message appeared:

“The Thread radio has multiprotocol enabled. To change channels when the Thread radio has multiprotocol enabled, use the hardware settings menu.”

How I can change the channel? Where I can find the “hardware settings menu”?
Thanks

**Comment by darkxst** [comment]
Hi Solarflor,
  Only solution to this is change the Thread Channel via REST Api see this link for details:
  https://github.com/darkxst/silabs-firmware-builder/wiki/RCP-Multipan-Channel-Mismatch
  
  However Multipan support in Home Assistant is being deprecated and it is now recommended to use a separate dongle dedicated to running Thread (OTBR Addon).

**Reply by Solarflor** [reply]
Many thanks
